src/main.ts

```diff
@@ -3,7 +3,7 @@ import App from './App.vue'
 import Hotjar from 'vue-hotjar'
 import { getVueRouter } from '@/router'
 import { getVuexStore } from '@/store'
-import { getConfig, getVuetify, initLdClient } from '@/plugins'
+import { getConfig, getVuetify, initLdClient, isSigningIn, isSigningOut } from '@/plugins'
```

Comment:
This should not show as a change in your commit. I think you did not rebase correctly.

Again, this appears correct so don't worry about it right now, but you will need to figure out rebasing for next time :)